docs/concepts/client/index.md

```diff
@@ -148,16 +148,6 @@ Client(host='https://my.awesome.flow:1234', port=4321)
 ```
 ````
 
-````{admonition} Caution
-:class: caution
```

Comment:
This should be kept, just mention that we use an RLock so that the effect they expect may not be observed